### PR TITLE
 minor fix to make sesheta approver work with zuul ci

### PR DESCRIPTION
**Pull request by harshad16**
PR Fixes the issue as sesheta was not checking for zuul ci. ;)

### CODE REVIEW

File: sesheta/common.py
Comment (harshad16) [inline sesheta/common.py:35]: should this change to "local/gate" from "local/check" !?

Comment (harshad16) [reply sesheta/common.py:35]: I thought thoth-zuul would merge the PR if approved label is added without waiting for reviewer approval, but it actually waits for reviews review.
"local/check" fits best to add labels based upon its results.

**Review by goern — APPROVED** [review]

**Pull request merged**
